src/styles/header.scss

```diff
@@ -0,0 +1,60 @@
+.header {
+  width: 100%;
+  padding: 0 50px;
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  box-sizing: border-box;
+
+  &__nav {
+    &__logo-img {
+      display: flex;
+      height: 40px;
+      width: 40px;
+    }
+
+    &-list {
+      display: flex;
+      list-style-type: none;
+      margin: 0;
+      padding: 0;
+    }
+
+    &-item {
+      margin-right: 20px;
+    }
+
+    &-item:last-child {
+      margin-right: 0;
+    }
+
+    &-link.is-active {
+      color: var(--hover-color);
+      position: relative;
+    }
+
+    &-link {
+      display: flex;
+      line-height: 60px;
```

Comment:
The `line-height` property should be relative to the font size, not a fixed value. It's better to use unitless values for `line-height`, which will calculate the line height relative to the font size of the element. For example, `line-height: 1.5;`